src/index.js

```diff
@@ -23,84 +23,84 @@ class MyArray {
       return undefined;
     }
 
-    const a = this[this.length - 1];
+    const lastElement = this[this.length - 1];
     delete this[this.length - 1];
     this.length -= 1;
 
-    return a;
+    return lastElement;
   }
 
   forEach(callback, thisArg) {
-    const mas = this;
+    const massif = this;
```

Comment:
You don't need const massiv at all, use this.

Reply:
Massif? Mas? What about array?